lib/JitEngine.hpp

```diff
@@ -69,6 +73,33 @@ class JitEngine {
   } Config;
 };
 
+inline
+SmallVector<RuntimeConstant, 8>& getPendingJitVariables() {
+  static SmallVector<RuntimeConstant, 8> PendingJitVariables;
+  return PendingJitVariables;
+}
+
+inline
+DenseMap<StringRef, SmallVector<RuntimeConstant, 4>>& getJitVariableMap() {
+  static DenseMap<StringRef, SmallVector<RuntimeConstant, 4>> JitVariableMap;
```

Comment:
is it safe to use `StringRef` for the key without the ownership? I assume yes because it's a ref to the global we create in the ProteusPass? Add comment